src/main/scala/com/cognite/spark/datasource/DataPointsRelation.scala

```diff
@@ -1,5 +1,7 @@
 package com.cognite.spark.datasource
 
+import java.util.concurrent.Executors
```

Comment:
Is this still being used?

Reply:
Nope! Couple of dead imports there, I'll remove them.